tests/integration/conftest.py

```diff
@@ -56,6 +56,53 @@ def apify_client_async() -> ApifyClientAsync:
     return ApifyClientAsync(api_token, api_url=api_url)
 
 
+class RunActorFunction(Protocol):
+    """A type for the `run_actor` fixture."""
+
+    def __call__(
+        self,
+        actor: ActorClientAsync,
+        *,
+        run_input: Any = None,
```

Comment:
Isn't `run_input` a dict?

Reply:
It really is `Any` - https://github.com/apify/apify-client-python/blob/v1.8.1/src/apify_client/clients/resource_clients/actor.py#L591

Reply:
Huh, weird, but not important.